code/lib/manager-api/src/modules/addons.ts

```diff
@@ -18,11 +19,19 @@ export interface SubAPI {
   /**
    * Returns a collection of elements of a specific type.
    * @protected This is used internally in storybook's manager.
+   * @template FAKE - The type of the elements in the collection.
```

Comment:
What's going on here?

Reply:
`getElements` was already generic, but I want to ignore that when passed in, because after my changes it's usually wrong.

Now it will return the correct type if you pass the correct `type` argument into it.
So to make `getElements` backwards compatible I discard the first type-argument.
I felt like being on the safe side of backwards compatibility here.

I could remove it and see if it's really needed.
`getElements` is sort-of an internal API only.

Reply:
I guess making a breaking change isn't kosher but maybe ok to make a breaking type change in this case? Make make it `| any` and drop it if it doesn't match the new type?